lib/actual_db_schema/patches/migration_context.rb

```diff
@@ -36,6 +43,45 @@ def migration_files
         current_branch_file_names = current_branch_files.map { |f| ActualDbSchema.migration_filename(f) }
         other_branches_files.reject { |f| ActualDbSchema.migration_filename(f).in?(current_branch_file_names) }
       end
+
+      def status_up?(migration)
+        migrations_status.any? do |status, version|
+          status == "up" && version.to_s == migration.version.to_s
+        end
+      end
+
+      def prompt_user_for_migration(migration)
+        loop do
+          print "Do you want to rollback this migration? [y,n] "
```

Comment:
```suggestion
          print "Rollback this migration? [y,n] "
```

Reply:
@VladislavSokov please make a screenshot of how this process of input looks like. Does it print the migration content or at least it's name? It could also print info from which branch those phantom migration comes from. That would be even more informative. We can add a separate task for that if that makes sense.

Reply:
![image](https://github.com/widefix/actual_db_schema/assets/62177364/90b2ed4a-dc55-4f11-b323-2f68d5f71cb0)

Reply:
We should add some intro text before printing the migration code explaining why it's printed.